### ISSUE TITLE
Log training loss

### ISSUE THREAD

**Issue by MaxiBoether** [eth-easl/modyn#488]
We should at the trainer server (optionally) collect the training loss as well and put it into the trainer server log. this might help to understand performance on the training set without requiring to eval on it.

**Comment by XianzheMa** [comment]
on it
